src/flint/types/dirichlet.pyx

```diff
@@ -189,6 +189,9 @@ cdef class dirichlet_char(object):
         acb_dirichlet_l((<acb>v).val, (<acb>s).val, self.G.val, self.val, getprec())
         return v
 
+    # For backwards compatibility we allow self.l(s) see Issue #210
+    l = l_function # no-cython-lint
```

Comment:
There isn't any point in doing this unless the docstring is updated to suggest calling the new function name.

I would do it like:
```
def l_function(self, s):
    """
    >>> l_function()
    123
    """
    ...


def l(self, s):
    """
    Alias for :meth:`l_function`
    """
    return self.l_function(s)
```

Reply:
oh sorry, i just missed this